modules/apps/27-interchain-accounts/host/keeper/relay_test.go

```diff
@@ -309,6 +334,27 @@ func (suite *KeeperTestSuite) TestOnRecvPacket() {
 			},
 			false,
 		},
+		{
+			"unauthorised: message type not allowed",
```

Comment:
maybe add a comment that we explicitly don't update the host params

Reply:
Added!